cypress/e2e/shared/tasks.test.ts

```diff
@@ -747,3 +717,12 @@ function createFirstTask(
   cy.getByTestID('task-form-schedule-input').type(interval)
   cy.getByTestID('task-form-offset-input').type(offset)
 }
+
+const clearAndType = (testId: string, value: string | number) => {
```

Comment:
nit: potentially renaming this to `getByTestIdAndSetInputValue` so that we know exactly what this is doing & what parameters it's expecting